src/components/About.astro

```diff
@@ -16,14 +16,15 @@ const images = [
       <div class="flex flex-col">
         <div class="flex-grow">
           <h2 class="text-4xl text-dark-green lg:text-6xl">
-            Finally, premium coworking space in Middleburg
+            Finally, premium coworking space in the heart of historic Middleburg
```

Comment:
```suggestion
            Premium coworking space in the heart of historic Middleburg
```

@chuckcarpenter I think this gets a little long for the heading, but I do like your suggestion. Perhaps we can remove `finally` to shorten it a tiny bit?